Resources/Prototypes/Species/species_weights.yml

```
@@ -6,7 +6,8 @@
     Reptilian: 4
     SlimePerson: 3
     Oni: 3 #Nyanotrasen Oni, see Prototypes/Nyanotrasen/Entities/Mobs/Species/Oni.yml
-    Felinid: 4 # Nyanotrasen - Felinid, see Prototypes/Nyanotrasen/Entities/Mobs/Species/felinid.yml
+    # Felinid: 4 # Nyanotrasen - Felinid, see Prototypes/Nyanotrasen/Entities/Mobs/Species/felinid.yml
     Vulpkanin: 4 # DeltaV - Vulpkanin, see Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml
     Diona: 2
     Shadowkin: 0
+    Tajaran: 3
```

```suggestion
    Tajaran: 3 # Einstein Engines, see <...yml>
```
That and also all of the Tajaran paths are fucked. Move them all to `EinsteinEngines/` where uncategorized.

---

Same as above